Formula/rawtoaces.rb

```diff
@@ -0,0 +1,26 @@
+class Rawtoaces < Formula
+  desc "RAW to ACES Utility"
+  homepage "https://github.com/ampas/rawtoaces"
+  url "https://github.com/ampas/rawtoaces/archive/v1.0.tar.gz"
+  sha256 "9d15e7e30c4fe97baedfdafb5fddf95534eee26392002b23e81649bbe6e501e9"
+
+  depends_on "cmake" => :build
+  depends_on "libraw"
+  depends_on "ilmbase"
+  depends_on "aces_container"
+  depends_on "boost"
+  depends_on "ceres-solver"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "test"
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match /convert RAW digital camera files to ACES/, shell_output("#{bin}/rawtoaces --help", 255).strip
```

Comment:
Would be nice not to bother with a regex since we're just checking a simple string. A `""` will suffice.

Reply:
Hi, Thanks for your feedback! 

I am new to writing ".rb" file for the new formula, would you mind elaborating on the "regex" part? Are you referring to the "assert_match" in the "test" section? Thanks!

Reply:
@miaoqi Dom means `"convert RAW digital camera files to ACES"` instead of `/convert RAW digital camera files to ACES/`.

But this needs to be a test that exercises and verifies the functionality of the software.

Reply:
Hi, any feedback on the updated formula? Thanks a lot!

Reply:
> But this needs to be a test that exercises and verifies the functionality of the software.

I think this still needs addressing.

Reply:
Hi, thank you for your timely feedback! I appreciate it.

I have just made the 3rd commit. It includes an exercise that verifies the output of a functionality. :)

Please let me know if it looks good to you.

Thanks a lot!